api/resources/views/job-form/show.blade.php

```diff
@@ -0,0 +1,23 @@
+@extends('layouts.app')
+
+@section('content')
+
+    <div class="container mt-4">
+        @include('errors')
+        <form action="{{ route('filled-forms.store') }}" method="post">
```

Comment:
The inputs reset after failed attempt to submit. Fix that, terrible UX.

Reply:
Okay

Reply:
Done